skops/card/_model_card.py

```diff
@@ -172,7 +210,7 @@ class Card:
 
     Parameters
     ----------
-    model: estimator object
+    model: pathlib.path, str, or sklearn estimator object
         Model that will be documented.
```

Comment:
Could you please extend the docstring, now that we have more options here?

Reply:
i changed to this "pathlib.path, str, or sklearn estimator object"
what should we add more ?

Reply:
How about something along the lines that if a str or Path is passed, the model will be loaded. This could be surprising behavior for some users and should be mentioned.